lib/pam/pam.go

```diff
@@ -2,9 +2,9 @@ package pam
 
 import (
 	"errors"
-	"fmt"
 	"github.com/intel/rmd/lib/pam/config"
 	"github.com/msteinert/pam"
+	"github.com/sirupsen/logrus"
```

Comment:
How about using log "github.com/sirupsen/logrus" just like other model does.

Reply:
Looks better. I have made the changes to the PR.